src/runtime/config/configuration-clh.toml.in

```diff
@@ -38,6 +38,13 @@ image = "@IMAGEPATH@"
 # disable applying SELinux on the VMM process (default false)
 disable_selinux=@DEFDISABLESELINUX@
 
+# disable applying SELinux on the container process
+# If set to false, the type `container_t` is applied to the container process by default.
+# Note: To enable guest SELinux, the guest rootfs must be CentOS that is created and built
+# with `SELINUX=yes`.
+# (default: true)
+disable_guest_selinux=@DEFDISABLEGUESTSELINUX@
```

Comment:
This block should be also propagated to the CCv0 specific configurations,

Reply:
@GeorginaKin - are you able to mirror these changes across? Let me know if you need any help

Reply:
I've added the code block to the following files `configuration-clh-tdx.toml.in`, `configuration-qemu-sev.toml.in`, `configuration-qemu-tdx.toml.in`. Is there anymore that need it?

Reply:
You covered it all, @GeorginaKin, thanks!